app/src/main/java/edu/agh/susgame/front/gui/components/menu/components/lobby/elements/LobbyComp.kt

```diff
@@ -51,7 +52,8 @@ import kotlinx.coroutines.launch
 
 @Composable
 fun LobbyComp(
-    lobbyInit: Lobby,
+    lobbyDetails: LobbyDetails,
+    lobbyPin: LobbyPin?,
```

Comment:
> czy jest opcja, żeby przekazać ten pin gry do widoku lobby żeby móc go tam wyświetlić? ktoś może je zapomnieć a chyba w lobby można go wyświetlić

Tutaj masz info o pinie obecnej gierki do dyspozycji, śmiało dodaj to do widoku, bo faktycznie poprawi UX